### PR TITLE
Add my app fr.fgrabenstaetter.DigitalAssets

### PR DESCRIPTION
**Pull request by fgrabenstaetter**

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build fr.fgrabenstaetter.DigitalAssets

**Comment by fgrabenstaetter** [comment]
I have fixed the appdata file

**Comment by nedrichards** [comment]
@fgrabenstaetter you can kickoff the test builds yourself if you want as well.

bot, build fr.fgrabenstaetter.DigitalAssets

File: fr.fgrabenstaetter.DigitalAssets.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "cleanup-commands": [ "glib-compile-schemas /app/share/glib-2.0/schemas" ],
+    "modules": [
+        {
+            "name": "digital-assets",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
```
Comment (nedrichards) [inline fr.fgrabenstaetter.DigitalAssets.json]: this needs to have a specific pinned release.

**Comment by fgrabenstaetter** [comment]
bot, build fr.fgrabenstaetter.DigitalAssets

**Comment by barthalion** [comment]
bot, build fr.fgrabenstaetter.DigitalAssets

**Comment by barthalion** [comment]
bot, build fr.fgrabenstaetter.DigitalAssets

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/fr.fgrabenstaetter.DigitalAssets

Thanks!

**Pull request closed without merging**